src/packages/verticaltabs/verticaltabs.taro.tsx

```
@@ -0,0 +1,292 @@
+  }
+
+  useEffect(() => {
+    let index = titles.current.findIndex(
+      (t) => String(t.value) === String(value)
+    )
+    index = index < 0 ? 0 : index
+    scrollIntoView(index)
+  }, [value])
+
+  const tabChange = (item: TabsTitle, index: number) => {
+    onClick && onClick(item.value)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)